libs/viewer/src/SimpleViewer.cpp

```diff
@@ -1645,7 +1647,13 @@ void SimpleViewer::updateUserInterface() {
     // At this point, all View settings have been modified,
     //  so we can now push them into the Filament View.
     applySettings(mSettings.view, mView);
-    mIndirectLight->setIblOptions(mSettings.lighting.iblOptions);
+
+    // Set IBL options
+    {        
+        filament::math::float3 linearBgColor = filament::Color::toLinear(mSettings.viewer.backgroundColor);
+        mSettings.lighting.iblOptions.iblTintAndIntensity.rgb = linearBgColor;
+        mSettings.lighting.iblOptions.iblTintAndIntensity.a = mSettings.lighting.iblOptions.iblTintAndIntensity.w;
```

Comment:
isn't this line a noop?

Reply:
Nice catch, it indeed was! Fixed in https://github.com/shapr3d/filament/pull/50/commits/ca9e7a290eaa87fbc3aea863f4f188a8f900a25d